packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -0,0 +1,124 @@
+import utils from '@percy/sdk-utils';
+import tmp from 'tmp';
+import fs from 'fs/promises';
+
+import CommonMetaDataResolver from '../metadata/commonMetaDataResolver.js';
+import Tile from '../util/tile.js';
+import Driver from '../driver.js';
+
+const log = utils.logger('webdriver-utils:genericProvider');
+// TODO: Need to pass parameter from sdk and catch in cli
+const CLIENT_INFO = 'local-poc-poa';
```

Comment:
we already collect the information from SDK, you need to pass that on.

Reply:
Yes, this will be covered in `v2` update.

Reply:
we won't be releasing till then right?